### ISSUE TITLE
 Module build failed: Error: Couldn't find preset "es2015" relative to directory 

### ISSUE THREAD

**Issue by tsankuanglee** [ratiw/vuetable-2-tutorial#27]
Had to do

> $ yarn add babel-preset-es2015

to make it work.

Could you add that to the installation guide or dependency?

**Comment by ratiw** [comment]
@tsankuanglee It's already there in the [`package.json`](https://github.com/ratiw/vuetable-2-tutorial/blob/master/package.json#L23). The `package.json` was automatically generated by `vue-cli`, so I think it should be installed by default.

**Comment by tsankuanglee** [comment]
You are right. I have no idea what happened. I tested again and it worked this time.

Thanks for the tutorial!

**Comment by clemsontiger** [comment]
odd, i'm getting this too, even with it listed in devDependencies

**Comment by tsankuanglee** [comment]
@clemsontiger could it be our particular yarn bulid? I'm using Arch here.

**Comment by clemsontiger** [comment]
maybe, i ran the command as you did to get it to work:
``$ yarn add babel-preset-es2015``

**Comment by 1415926535** [comment]
It's looking for my preset es2015 up under node_module/vuetable-2. I would have thunk it would be under node_modules/ all proper and stuff. I have a feeling this is over my head but thought maybe it could help someone else figure something else out. 

I suspect my problem is I did something out of order. I know composered then the cp .env && art key:gen and then npm install. But then I started installing everything balked on. I've rm -rfed node_modules once. So now I just do straight `npm install`. It didn't work. It's balking on the first thing I mentioned up top there. If I'm in the Rhumsfeldian state where I don't know enough to know I don't know, then disregard.
